tests/testing_support/db_settings.py

```diff
@@ -147,6 +147,53 @@ def redis_cluster_settings():
     ]
     return settings
 
+def valkey_settings():
+    """Return a list of dict of settings for connecting to valkey.
+
+    Will return the correct settings, depending on which of the environments it
+    is running in. It attempts to set variables in the following order, where
+    later environments override earlier ones.
+
+        1. Local
+        2. Github Actions
+    """
+
+    host = "host.docker.internal" if "GITHUB_ACTIONS" in os.environ else "localhost"
+    instances = 2
+    settings = [
+        {
+            "host": host,
+            "port": 8080 + instance_num,
+        }
+        for instance_num in range(instances)
+    ]
+    return settings
+
+
+def valkey_cluster_settings():
```

Comment:
Same thing here-this isn't used anywhere.

Reply:
@hmstepanek removed.